File: BE-FindingJob/src/main/java/com/example/befindingjob/dto/employer/CompanyInfo.java
package com.example.befindingjob.dto.employer;

import com.example.befindingjob.dto.admin.UserInfo;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CompanyInfo {
    private int companyId;
    private String companyName;
    private UserInfo userInfo;
    private String address;
    private String logoUrl;
    private String website;
    private String description;
    private String createdAt;
    private String updatedAt;
}

File: JobHub/app/src/main/java/com/example/jobhub/dto/employer/CompanyInfo.kt
package com.example.jobhub.dto.employer

import com.example.jobhub.dto.admin.UserInfo

data class CompanyInfo (
    var companyId: Int? = null,
    var companyName: String = "",
    var userInfo: UserInfo? = null,
    var address: String = "",
    var logoUrl: String = "",
    var website: String = "",
    var description: String = "",
    var createdAt: String = "",
    var updatedAt: String = "",
)